README.md

```diff
@@ -30,13 +31,14 @@ pod 'InputBarAccessoryView', '4.2.2'
 ```ruby
 # Swift 5.0
 github "nathantannar4/InputBarAccessoryView"
+
 # Swift 4.2
 github "nathantannar4/InputBarAccessoryView" "4.2.2"
 ```
 
 ### Requirements
 
-iOS 9.0+
+iOS 11.0+
```

Comment:
I would leave it as iOS 9+ but where we have the Swift 4 pod version, add an entry for pre iOS 11 so people could checkout the last compatible version.